latch/types/file.py

```diff
@@ -54,9 +54,10 @@ def task(file: LatchFile):
     def __init__(
         self, path: Union[str, PathLike], remote_path: PathLike = None, **kwargs
     ):
-
         if remote_path is not None:
-            self._remote_path = LatchURL(remote_path).url  # validates url string
+            self._remote_path = remote_path
+        if _is_valid_url(path) and remote_path is None:
+            self._remote_path = path
```

Comment:
```suggestion
        elif _is_valid_url(path):
            self._remote_path = path
        else:
            raise ValueError("Invalid arguments to LatchFile")
```

or something similar

Reply:
this logic checks for 
* `LatchFile(<remote_path>)`
* `LatchFile(<local_path>, <remote_path>)`
but `LatchFile(<local_path>)` is still valid, so error should not be thrown